README.md

```diff
@@ -79,7 +79,7 @@ Add to your app's build.gradle:
 
 ```gradle
 dependencies {
-    implementation 'com.microsoft.identity.client:msal:2.0.+'
+    implementation 'com.microsoft.identity.client:msal:2.3.+'
```

Comment:
There is another PR by Brian to update this to 3.0+

Reply:
We need 4.0.2 now